charts/kubewarden-policy-server/templates/NOTES.txt

```diff
@@ -0,0 +1,11 @@
+kubewarden-policy-server installed.
+
+You can start defining cluster admission policies by using the
```

Comment:
we should explain here that you can now start defining `PolicyServers` not `ClusterAdmissionPolicies`